appengine/firebase-tictactoe/pom.xml

```diff
@@ -0,0 +1,121 @@
+<!--
+ Copyright 2015 Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <groupId>com.example.appengine</groupId>
+    <artifactId>appengine-firebase</artifactId>
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>doc-samples</artifactId>
+        <version>1.0.0</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <properties>
+        <objectify.version>5.1.13</objectify.version>
+    </properties>
+    <!-- [START set_versions] -->
+    <prerequisites>
+        <maven>3.3.9</maven>
+    </prerequisites>
+    <!-- [END set_versions] -->
+    <dependencies>
+        <dependency>
+            <groupId>com.google.appengine</groupId>
+            <artifactId>appengine-api-1.0-sdk</artifactId>
+            <version>${appengine.sdk.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
```

Comment:
I typically use this as it's very easy to use, but consider [Google GSON](https://github.com/google/gson).  I'm trying to switch.